cpp/include/cugraph/graph_view.hpp

```diff
@@ -59,10 +59,10 @@ class serializer_t;  // forward...
  *
  * To be more specific, a GPU with (col_comm_rank, row_comm_rank) will be responsible for
  * col_comm_size rectangular partitions [a_i,b_i) by [c,d) where a_i =
- * vertex_partition_offsets[row_comm_size * i + row_comm_rank] and b_i =
- * vertex_partition_offsets[row_comm_size * i + row_comm_rank + 1]. c is
- * vertex_partition_offsets[row_comm_size * col_comm_rank] and d =
- * vertex_partition_offsests[row_comm_size * (col_comm_rank + 1)].
+ * vertex_partition_range_offsets[row_comm_size * i + row_comm_rank] and b_i =
+ * vertex_partition_range_offsets[row_comm_size * i + row_comm_rank + 1]. c is
+ * vertex_partition_range_offsets[row_comm_size * col_comm_rank] and d =
+ * vertex_partition_range_offsets[row_comm_size * (col_comm_rank + 1)].
```

Comment:
Could we rename this to specify that it is a range of vertex ids?
vertex_id_ranges perhaps?

Reply:
To be more consistent with the rest, we may need to rename `vertex_partition_range` with `vertex_id_partition_range` but I feel like this is a bit too lengthy (I thought `vertex_partition` somewhat implies partitioning of vertex ID ranges but it seems like this implication is not clear enough).

I added additional comments below to make this point clearer. Do you think this is sufficient or we need to further discuss about renaming?